zookeeper-metrics-providers/zookeeper-prometheus-metrics/src/test/java/org/apache/zookeeper/metrics/prometheus/PrometheusMetricsProviderTest.java

```diff
@@ -166,9 +165,7 @@ public void testGauge() throws Exception {
         // override gauge, without unregister
         provider.getRootContext().registerGauge("gg", gauge0);
 
-        provider.dump((k, v) -> {
-            count[0]++;
-        }
+        provider.dump((k, v) -> count[0]++
         );
```

Comment:
Please don't do the unrelated refactor in this PR

Reply:
reverted